player-small.js

```diff
@@ -302,6 +302,18 @@ var CPlayer = function() {
         return mCurrentCol / mSong.numChannels;
     };
 
+    // Create a AudioBuffer from the generated audio data
+    this.createAudioBuffer = function(context) {
+        var buffer = context.createBuffer(2, mMixBuf.length / 2, 44100);
+        for (var i = 0; i < 2; i ++) {
+            var data = buffer.getChannelData(i);
+            for (var j = 0, k = i; j < buffer.length; j ++, k += 2) {
+                data[j] = mMixBuf[k] / 65536;
```

Comment:
Minification optimization:

```JavaScript
            for (var j = i; j < mNumWords; j += 2) {
                data[j >> 1] = mMixBuf[j] / 65536;
```

(untested, but I think it's right)

Reply:
Done! And tested.